component/web/security/src/main/resources/db/changelog/gatein-token.db.changelog-1.0.0.xml

```diff
@@ -30,4 +30,11 @@
   <changeSet author="gatein-token" id="1.0.0-2" dbms="oracle,postgresql">
     <createSequence sequenceName="SEQ_GATEIN_TOKEN_ID_GENERATOR" startValue="1"/>
   </changeSet>
+  <changeSet author="gatein-token" id="1.0.0-3">
+    <addColumn tableName="PORTAL_TOKENS">
+      <column name="TOKEN_TYPE" type="VARCHAR(250)">
+        <constraints nullable="false"/>
+      </column>
+    </addColumn>
+  </changeSet>
```

Comment:
This created a problem when you start with an existing database : 
- Get fresh 6.3.x-maintenance
- Start, create user root and make a forgot-password to request a token
- Ensure that the token is created in database
- Stop server
- deploy your fix
- start the server

There is an error in logs : 
`2022-01-13 13:57:00,308 | ERROR | db/changelog/gatein-token.db.changelog-1.0.0.xml: db/changelog/gatein-token.db.changelog-1.0.0.xml::1.0.0-3::gatein-token: Change Set db/changelog/gatein-token.db.changelog-1.0.0.xml::1.0.0-3::gatein-token failed.  Error: default expression needed in statement [ALTER TABLE PUBLIC.PORTAL_TOKENS ADD TOKEN_TYPE VARCHAR(250) NOT NULL] [Failed SQL: ALTER TABLE PUBLIC.PORTAL_TOKENS ADD TOKEN_TYPE VARCHAR(250) NOT NULL] [liquibase<localhost-startStop-7>] 
liquibase.exception.DatabaseException: default expression needed in statement [ALTER TABLE PUBLIC.PORTAL_TOKENS ADD TOKEN_TYPE VARCHAR(250) NOT NULL] [Failed SQL: ALTER TABLE PUBLIC.PORTAL_TOKENS ADD TOKEN_TYPE VARCHAR(250) NOT NULL]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:301)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:55)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:107)
	at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1251)
	at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1234)
	at liquibase.changelog.ChangeSet.execute(ChangeSet.java:554)
	at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:51)
	at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:73)
	at liquibase.Liquibase.update(Liquibase.java:212)
	at liquibase.Liquibase.update(Liquibase.java:192)
	at liquibase.Liquibase.update(Liquibase.java:188)
	at liquibase.Liquibase.update(Liquibase.java:181)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.applyChangeLog(LiquibaseDataInitializer.java:169)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.initData(LiquibaseDataInitializer.java:150)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.initData(LiquibaseDataInitializer.java:112)
	at org.exoplatform.commons.persistence.impl.LiquibaseDataInitializer.start(LiquibaseDataInitializer.java:99)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.exoplatform.container.LifecycleVisitor.traverse(LifecycleVisitor.java:103)
	at org.exoplatform.container.LifecycleVisitor.start(LifecycleVisitor.java:166)
	at org.exoplatform.container.ConcurrentContainer.start(ConcurrentContainer.java:472)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:67)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:67)
	at org.exoplatform.container.AbstractContainer.start(AbstractContainer.java:67)
	at org.exoplatform.container.ExoContainer.start(ExoContainer.java:309)
	at org.exoplatform.container.ExoContainer.start(ExoContainer.java:282)
	at org.exoplatform.container.RootContainer$7.run(RootContainer.java:881)
	at org.exoplatform.container.RootContainer$7.run(RootContainer.java:875)
	at org.exoplatform.commons.utils.SecurityHelper.doPrivilegedExceptionAction(SecurityHelper.java:311)
	at org.exoplatform.container.RootContainer.buildRootContainer(RootContainer.java:874)
	at org.exoplatform.container.RootContainer.getInstance(RootContainer.java:957)
	at org.exoplatform.web.GenericHttpListener.contextInitialized(GenericHttpListener.java:156)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4705)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5168)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:743)
	at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:719)
	at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
	at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:1015)
	at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1895)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.sql.SQLSyntaxErrorException: default expression needed in statement [ALTER TABLE PUBLIC.PORTAL_TOKENS ADD TOKEN_TYPE VARCHAR(250) NOT NULL]
	at org.hsqldb.jdbc.JDBCUtil.sqlException(Unknown Source)
	at org.hsqldb.jdbc.JDBCUtil.sqlException(Unknown Source)
	at org.hsqldb.jdbc.JDBCStatement.fetchResult(Unknown Source)
	at org.hsqldb.jdbc.JDBCStatement.execute(Unknown Source)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy13.execute(Unknown Source)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:299)
	... 46 common frames omitted
Caused by: org.hsqldb.HsqlException: default expression needed
	at org.hsqldb.error.Error.error(Unknown Source)
	at org.hsqldb.error.Error.error(Unknown Source)
	at org.hsqldb.TableWorks.addColumn(Unknown Source)
	at org.hsqldb.StatementSchema.getResult(Unknown Source)
	at org.hsqldb.StatementSchema.execute(Unknown Source)
	at org.hsqldb.Session.executeCompiledStatement(Unknown Source)
	at org.hsqldb.Session.executeDirectStatement(Unknown Source)
	at org.hsqldb.Session.execute(Unknown Source)
	... 55 common frames omitted
`